torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -420,13 +420,20 @@ def _get_compute_node(_binary_node, _other_index):
                 _compute_index = 1 if (_other_index == 0) else 0
                 return _binary_node.args[_compute_index]
 
-            if any(
-                len(
-                    _get_remaining_users(
-                        n.args[other_index], _get_compute_node(n, other_index)
+            def _is_other_input_not_inplace_changeable(_binary_node, _other_index):
```

Comment:
Would it be simpler?
```suggestion
            def _other_input_inplaceable(_binary_node, _other_index):
```

Reply:
Thanks for the suggestion and changed.